### ISSUE TITLE
The code does not work when I change the animation resource

### ISSUE THREAD

**Issue by zifenggao** [yandeu/phaser3-spine-example#1]
I change the mainScene.js file like that.Just replace the spine resources.But it not work,the spine animate no show.Kindly ask for help.

Here are the spine resources:https://pan.baidu.com/s/1kgii6Loa5yHgnYOIvqNx0w

```js
// import button from './button.js'

var goblin
let skins = ['goblin', 'goblingirl']
let attachments = ['spear', 'dagger', null]
let animations = ['walk', 'idle']

class MainScene extends Phaser.Scene {
  constructor() {
    super({ key: 'MainScene' })
  }

  preload() {

    this.load.setPath('spine/')
    // this.load.spine('goblin', 'goblins.json', 'goblins.atlas')
    this.load.spine('goblin', 'spineboy.json', 'spineboy.atlas')
  }

  create() {
    goblin = this.add.spine(400, 600, 'goblin')
  }
}

export default MainScene
```

**Comment by zifenggao** [comment]
It may be because my spine resource version is too low, it is version 3.6, and the SpinePlugin in the code supports version 3.8. Do you know that Phaser has a plugin that supports spine 3.6?

**Comment by zifenggao** [comment]
Solved, version 3.6 requires SpineWebGLPlugin in Phaser 3.18
